src/cryoemservices/cli/clem_align_and_merge.py

```diff
@@ -14,6 +14,71 @@
 from cryoemservices.wrappers.clem_align_and_merge import align_and_merge_stacks
 
 
+def path_or_none(value: str):
+    """
+    Parses the command line input, converting "null" into None, existing file paths
+    into Path objects, and returning None otherwise.
+    """
+    if value == "null":
+        return None
+    elif Path(value).exists():
+        return Path(value)
+    else:
+        print(
+            "The provided file path doesn't exist. Will attempt to find file using "
+            "the default settings"
+        )
+        return None
+
+
+def int_or_none(value: str):
+    """
+    Parses the command line input, converting "null" into None and returning an
+    integer otherwise.
+    """
+    if value.lower() == "null":
+        return None
+    try:
+        return int(value)
+    except ValueError:
+        raise argparse.ArgumentTypeError(
+            f"Invalid value provided: {value}. "
+            "Input must be either an integer or 'null'"
+        )
+
+
+def str_or_none(value: str):
+    """
+    Converts the "null" keyword into None, and returns the string as-is otherwise.
+    """
+    if value == "null":
+        return None
+    return value
+
+
+def parse_list_of_paths(values: list[str]):
+    if any((not isinstance(file, str) for file in values)):
```

Comment:
Is it necessary to recheck that these are strings? The type in the function is list[str]